x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/audit/AuditTrailService.java

```diff
@@ -222,4 +223,15 @@ public void runAsDenied(String requestId, Authentication authentication, RestReq
             }
         }
     }
+
+    @Override
+    public void explicitAccessEvent(String requestId, AuditLevel eventType, Authentication authentication, String action, String[] indices,
```

Comment:
for followup, I think we should remove this class. We only have a single audit trail now